pkg/dependency/parser/dotnet/core_deps/testdata/without-runtime.deps.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Would it be possible to add another testcase that references some of the commonly reported false positives and proves that those aren't flagged?  For example - a deps file listing a vulnerable version of `System.Net.Http` etc.  Here's a sample deps file: 
[testRuntimeDeps.deps.json](https://github.com/user-attachments/files/16254592/testRuntimeDeps.deps.json)

This was created with 
```
dotnet new console -f net6.0
dotnet add package NETStandard.Library --version 1.6.0
dotnet build
```

This shows that all the old packages (some vulnerable) making up the NETStandard surface area are all excluded when targeting the latest frameworks which provide inbox support for all those package.